src/components/item/Container.test.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ItemContainer from './Container.js';
+import { cleanup, render, waitForElement } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+afterEach(cleanup);
+
+it('renders the correct item data', async () => {
+  const nock = require('nock');
+  const component = render(<ItemContainer match={{ params: { id: 1 } }} />);
+
+  const itemData = {
+    "data": {
+      "description": "test description",
+      "end_date": "2019-07-17T16:53:52Z",
+      "id": 1,
+      "image": "test-image-url",
+      "price": 10,
+      "title": "test title",
+    }
+  };
+
+  nock('http://localhost')
+    .get('/api/items/1')
+    .reply(200, itemData)
+
+  await waitForElement(() => component.getByTestId('item-1'));
+
+  component.getByAltText('test title')
+  component.getByText('test description');
+  component.getByText('2019-07-17T16:53:52Z');
+  component.getByText('Current Bid: $10');
+  component.getByText('test title');
```

Comment:
I'm a little unfamiliar with this test, but what does this assert? Just that the particular text exists in the component?